diff -ruN pam/meson.build pam.patch/meson.build
--- pam/meson.build	2024-10-24 11:00:00.000000000 +0300
+++ pam.patch/meson.build	2025-03-27 00:16:09.494807788 +0200
@@ -251,7 +251,7 @@
 if not libcrypt.found()
   libcrypt = cc.find_library('crypt')
 endif
-crypt_h = cdata.get('HAVE_CRYPT_H') == 1 ? '''#include <crypt.h>''' : '''#include <unistd.h>'''
+crypt_h = 1 == 1 ? '''#include <crypt.h>''' : '''#include <unistd.h>'''
 foreach f: ['crypt_r']
   if cc.has_function(f, prefix: crypt_h,
                      args: '-D_GNU_SOURCE',
